lunavl/sdk/indexes/stored_index.py

```diff
@@ -0,0 +1,115 @@
+"""Module realize dynamic and dense index."""
+from enum import Enum
+from pathlib import Path
+from typing import List, Optional
+
+from lunavl.sdk.descriptors.descriptors import BaseDescriptor, BaseDescriptorBatch
+from lunavl.sdk.errors.errors import LunaVLError
+from lunavl.sdk.errors.exceptions import LunaSDKException
+from .base import CoreIndex, IndexResult
+
+
+class IndexType(Enum):
+    """Available index type to save|load."""
+
+    # dense index
+    dense = "dense"
+    # dynamic index
+    dynamic = "dynamic"
+
+
+class DynamicIndex(CoreIndex):
+    """
+    Dynamic Index
+    """
+
+    @property
+    def descriptorsCount(self):
+        """Get actual count of descriptor in internal storage."""
+        return self._coreIndex.countOfIndexedDescriptors()
+
+    def append(self, descriptor: BaseDescriptor) -> None:
+        """
+        Appends descriptor to internal storage.
+        Args:
+            descriptor: descriptor with correct length, version and data
+        Raises:
+            LunaSDKException: if an error occurs while adding the descriptor
+        """
+        error = self._coreIndex.appendDescriptor(descriptor.coreEstimation)
```

Comment:
i think that need check a version of descriptor

Reply:
added and opened fsdk issue